### PR TITLE
Tech chair's content

### PR DESCRIPTION
**Pull request by stefan-balke**

### CODE REVIEW

File: ISMIR2021.tex [hunk 1]
```diff
@@ -151,12 +155,86 @@ \section{Introduction}
...
+\end{figure}
+
+Since ISMIR 2021 was fully digital, the conference platforms are the most essential part where all things glue together and most of the ``magic'' happens.
+
+\begin{itemize}
+    \item \textbf{Midspace:} Midspace is an all-in-one solution that provides a centralized web application to manage useful to schedule the conference and to assign respective roles to different sessions. For example, authors or session shares can be made fully responsible for their respective session content, sending reminders, let authors upload content etc. Figure~\ref{fig:midspace} shows excerpts from Midspace's graphical user interface. We decided to choose midspace as our centralized platform to orchestrate the conference. Midspace integrates a chat system, a video conferencing system (both async for keynotes and live for posters), a conference schedule/calendar and a system to let authors upload their content and videos themselves. This, midspaces replaced Slack (chat), miniconf (schedule website) and Dropbox (authors video upload). Furthermore, one of the main reasons where that we offered full support before and during the conference from the midspace team.
```
Comment (alexanderlerch) [inline ISMIR2021.tex]: "web application to manage useful to schedule the conference": rephrase

Comment (alexanderlerch) [inline ISMIR2021.tex]: "session shares": session chairs?

Comment (alexanderlerch) [inline ISMIR2021.tex]: "This, midspaces": replace with Thus, midspace ?

Comment (alexanderlerch) [inline ISMIR2021.tex]: "replaced Slack (chat), miniconf (schedule website) and Dropbox (authors video upload)" add reference to previous year's conference to explain why the word 'replace' is used?

Comment (alexanderlerch) [inline ISMIR2021.tex]: "that we offered full support": shouldn't this be that we WERE offered full support ?

File: ISMIR2021.tex [hunk 2]
```diff
@@ -151,12 +155,86 @@ \section{Introduction}
...
+    \item This required attempting to collect the necessary data from various sources as early as possible (to leave room for errors).
+    \item To make providing data as easy as possible for the other chairs, we attempted to support simplified data import formats, instead of the more comprehensive, original Midspace formats.
+    \item Once the conference start was very close, we switched over to manual data entry.
+    \item Because the registration system (which was not chosen by the tech chairs) did not offer a suitable API, authors and attendees had to be imported into Midspace manually by running custom-coded scripts.
+    \item The gathe.town map was created in a long process together with a graphic designer from korea that we payed for via Fiverr. The designer created the concept and designed the space with respect to our requirements (exptected number of attendees, etc.). He also helped us to fill in the sponsor area in gather town which resulted in many additional changes due to sponsor re-requests.
```
Comment (alexanderlerch) [inline ISMIR2021.tex]: replace "gathe.town"  and "gather town" with gather.town

File: ISMIR2021.tex [hunk 3]
```diff
@@ -151,12 +155,86 @@ \section{Introduction}
...
+    \item Events on \emph{Gater.town} were also manually entered via the map-making tool. This is a tedious process and can not be automated or synced to the conference schedule.
```
Comment (alexanderlerch) [inline ISMIR2021.tex]: \emph{Gater.town}: use consistent formatting through-out the text

File: ISMIR2021.tex [hunk 4]
```diff
@@ -151,12 +155,86 @@ \section{Introduction}
...
+    \item The main task of the tech-volunteers was to help session chairs and authors to seamlessly navigate the conference platforms. The tech-chairs were only a stand-by option for worst case scenarios---when admin access was required to make changes to the platform or when the tech-volunteer couldn't make it to the session.
+    \item As tech-volunteers we recruited 3-4 students (see~\ref{sssec:vol}) across each of the three timezones, that roughly correspond to Asia, Europe and US timezones. We had 12 volunteers in total.
+    \item A couple of weeks before the conference, all tech-volunteers were given a walkthrough of the conference platforms and a live demo of how to manage the platforms.
+    \item During the conference, the respective tech-chairs would remind the responsible tech-volunteers of the sessions a day earlier. 
+\end{itemize}
+
+\subsection{Lesson's Learned}
```
Comment (alexanderlerch) [inline ISMIR2021.tex]: "Lesson's Learned": -> Lessons Learned

File: ISMIR2021.tex [hunk 5]
```diff
@@ -151,12 +155,86 @@ \section{Introduction}
...
+
+\begin{itemize}
+    \item Negotiation of contracts, especially in the US, seem to have been way underestimated by us. Start that early, if things go bogus, include the board, they are more than willing to help out.
```
Comment (alexanderlerch) [inline ISMIR2021.tex:225]: maybe rephrase. The board was willing to help, but they certainly would have preferred to not be involved.

File: ISMIR2021.tex [hunk 6]
```diff
@@ -151,12 +155,86 @@ \section{Introduction}
...
+    \item Poster sessions in midspace were not a great experience---with the user having to enter and exit video calls of each of the poster. Consider organising the poster sessions in Gathertown in the future, which allows seamless navigation in the poster area, allowing the participants to talk with the nearby participants and authors.
+    \item One of the arguments for people to use Zoom for video conferencing in our community is the ability to easily share system audio in good quality. We noticed, however, that only a minority of users actually need this feature or does not know how to install a loopback software driver. Furthermore, the chrome browser has support to share system audio via the browser (at least on Windows). Thus, Zoom for audio based presentation isn't a hard requirement anymore. Given that we didn't had many issues with midspaces video system for either the keynotes nor the poster sessions (almost no dropouts), we can only suggest to not use zoom as additionaly platform when its not absolutely required. This also significalty reduces the organizational overhead as Zoom's backend doesn't allow to control the full schedule from multiple moderator accounts.
```
Comment (alexanderlerch) [inline ISMIR2021.tex]: "didn't had": -> didn't have

**Review by alexanderlerch — CHANGES_REQUESTED** [review]
Please see my comments for minor changes.

Do we have participation data that we can include?

**Comment by faroit** [comment]
all addressed except [this one](https://github.com/ismir2021/conferencereport/pull/2#discussion_r855499998) - i have no opinion on that.

@alexanderlerch you can use _suggestions_ when reviewing PRs on github for simple things like typos (See [(7) in this doc](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review). That saves a lot of time for everyone as the proposed changed can be merged in with a single click.
